host/bcm283x/clock.go

```diff
@@ -284,10 +284,10 @@ func (c *clock) setRaw(ctl clockCtl, div int) error {
 	}
 	d := clockDiv(div << clockDiviShift)
 	c.div = clockPasswdDiv | d
-	Nanospin(10 * time.Nanosecond)
+	Nanospin(10 * time.Microsecond)
```

Comment:
I forget why I had chosen this value. 2x 10µs seems a bit slow though so it be great if we could figure out which lowest value we could use. If you can assert that 10ns fails, we should at least add a TODO to clarify that we don't know the exact lower bound needed.

Reply:
I thought it was typo because 10µs is used everywhere else. Looks like it was intentional, I reverted them and it seems working so far.